fnl/lightspeed.fnl

```diff
@@ -351,18 +359,21 @@ character instead."
   (if (or ?target-windows omni?)
       (each [_ win (ipairs (or ?target-windows
                                [(. (vim.fn.getwininfo (vim.fn.win_getid)) 1)]))]
-        (vim.highlight.range win.bufnr hl.ns hl.group.greywash
-                             [(dec win.topline) 0] [(dec win.botline) -1] :v false
-                             hl.priority.greywash))
+        (hl-compat win.bufnr hl.ns hl.group.greywash
+                   [(dec win.topline) 0] [(dec win.botline) -1]
+                   {:regtype :charwise
```

Comment:
Charwise should be `:c` or `:v`, according to `:h setreg()`, shouldn't it? (Not that it matters too much at the moment, as `vim.highlight.range` calls `vim.region`, which [apparently](https://github.com/neovim/neovim/blob/f0ace6d419380907b719edd2ea1d2256e92beb84/src/nvim/lua/vim.lua#L441) only checks for `regtype:byte() == 22` (`^V`), but still...)